Enterprise-CMCS/macpro-mdct-mfp
# Add e2e approve Work Plan

### Description

- Add Playwright test for Admin user to approve a Work Plan
- Move and refactor WP create and SAR create tests for consistent format
- Refactor helpers to be more DRY
- Remove redundant login tests
- Update deploy to pass env variables to Playwright for seed script

### Related ticket(s)


CMDCT-3864

---
### How to test

1. Locally, run `yarn test:e2e` or `yarn test:e2e-ui` for interactive tests
2. Verify all tests pass






---
### Pre-review checklist
